test/validate_v1_v2_equality.ts

```diff
@@ -248,4 +259,108 @@ describe('Printings v1/v2 equality', () => {
   it('stripped_printed_text matches', () => {
     validate('stripped_text', 'stripped_printed_text');
   });
-});
\ No newline at end of file
+});
+
+describe('MWLs v1/v2', () => {
+  const standardRestrictions = getRestrictionsV2Json()['standard'];
+  const mwls = getMwlJson();
+  const pairs = mwls.map(mwl => [mwl, standardRestrictions.find(r => r.name == mwl.name)]);
+
+  const printings = getPrintingsV2Json ();
+  const printingsById = new Map<string, any>();
+  printings.forEach(p => {
+    printingsById.set(p.id, p);
+  });
+  const printingsByCardId = new Map<string, any>();
+  getCardsV2Json ().forEach(c => {
+    printingsByCardId.set(c.id, []);
+  });
+  printings.forEach(p => {
+    printingsByCardId.get(p.card_id).push(p);
+  });
+
+  function v1ToBanned(mwl) {
+    const arr: [string, Record<string, number>][] = Object.entries(mwl.cards);
+    return arr.filter(card => card[1].deck_limit != undefined && card[1].deck_limit == 0)
+              .map(card => card[0]).sort();
+  }
+  function v1ToRestricted(mwl) {
```

Comment:
all of the prep functions here would benefit from some comments.  I had to read a couple of times to clearly picture what the output was here.